### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by aspdye**
Updated the "And the works for 0.2" Part. Added "Unified Inbox" and "Mobile Screen" to "Why is it so awsome:".

### CODE REVIEW

**Comment by ChristophWurst** [comment]
@jancborchardt

**Comment by Gomez** [comment]
Looks good. Thanks @aspdye

**Comment by aspdye** [comment]
you're welcome :)

**Comment by jancborchardt** [comment]
Nice, thank you! :) Just some details:

* I would only not put the specific milestone / release number in there because that might change often.
* The unified inbox point can just be part of the first point
* I wouldn’t specifically mention the mobile compatibility since people expect it to work, and on mobile usually folks use native apps anyway. :)

**Comment by aspdye** [comment]
@jancborchardt Improved Point 1 and 3. 

Point 2: Maybe the unified inbox should be part of "You can use multiple mail accounts! Personal and company account? No problem."
Should we remove "Unified inbox" or integrate that in this point?

**Comment by aspdye** [comment]
@ChristophWurst @jancborchardt do you have an good New Screenshot?

**Comment by ChristophWurst** [comment]
> @ChristophWurst @jancborchardt do you have an good New Screenshot?

@jancborchardt has to fix the layout first (sidebar background)

**Comment by aspdye** [comment]
@jancborchardt Take a look ;)

**Comment by aspdye** [comment]
Think we could "merge" (It is called merge, or? ;)) this ;) 

(Dont know whether I have to do anything. I am new in GitHub things ;) )

**Comment by Gomez** [comment]
I have some more stuff:
- [ ] Remove the "dedicated email server" dream stuff. 
- [ ] Update Screenshot
- [ ] Move Maintainer Jakob Sack  to active (YEAH!)
- [ ] Move Troubleshooting to the bottom 
- [ ] Developer setup info: Add git clone to the install: "https://github.com/owncloud/mail.git"

@aspdye Would you help with this? Otherwise i can hijacking the PR. THX

**Comment by aspdye** [comment]
Will do this in 30 minutes. Thank you @Gomez

**Comment by aspdye** [comment]
Does someone have "good" new screenshots with some contact-pictures?

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Code Coverage](https://scrutinizer-ci.com/g/owncloud/mail/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/owncloud/mail/?branch=master)
 [![Dependency Status](https://www.versioneye.com/user/projects/54e50fadd1ec5734f400078a/badge.svg?style=flat)](https://www.versioneye.com/user/projects/54e50fadd1ec5734f400078a)
 
-**An email app for [ownCloud](https://owncloud.org) (minimum version 7.0.4 & PHP 5.4). At the moment it is a basic IMAP client and in heavy development.** (A possibility for the future would be to also have it be a dedicated email server. But that would always be optional, require additional dependencies and is out of scope for now.)
+**An email app for [ownCloud](https://owncloud.org) (minimum version 7.0.4 & PHP 5.4).**
```
Comment (jancborchardt) [inline README.md]: @Gomez why remove the IMAP clarification? I’d say this is an important info.

**Comment by jancborchardt** [comment]
@aspdye can you squash the commits into one? Then I will merge this.

Please no further additions for now – something for a separate pull request.

File: README.md (hunk 1 above)
Comment (Gomez) [inline README.md]: This should stay: "At the moment it is a basic IMAP client". Or something like this:
Its a basic IMAP client which integrates nicely with other ownCloud Apps. 
The "dedicated email server" should go. We will never add it.

Comment (jancborchardt) [inline README.md]: @aspdye can you change it to say after that (not in bold): »A basic IMAP client which integrates nicely with other ownCloud apps.«

**Comment by ChristophWurst** [comment]
@aspdye could you please squash those commits into one? :-)

**Comment by ChristophWurst** [comment]
@aspdye ping

**Comment by ChristophWurst** [comment]
replaced by #1081

**Pull request closed without merging**